src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -386,22 +386,23 @@ bool IsInstanceMemberReference(IOperation operation)
             if (typeInfo.Type?.SpecialType == SpecialType.System_String &&
                 typeInfo.ConvertedType?.IsFormattableString() == true)
             {
-                return typeInfo.ConvertedType;
+                return typeInfo.ConvertedType.WithNullability(typeInfo.Nullability.FlowState);
             }
 
             if (typeInfo.Type != null)
             {
-                return typeInfo.Type;
+                return typeInfo.Type.WithNullability(typeInfo.Nullability.FlowState);
             }
 
             if (typeInfo.ConvertedType != null)
             {
-                return typeInfo.ConvertedType;
+                return typeInfo.ConvertedType.WithNullability(typeInfo.ConvertedNullability.FlowState);
             }
 
             if (objectAsDefault)
             {
-                return semanticModel.Compilation.GetSpecialType(SpecialType.System_Object);
+                return semanticModel.Compilation.GetSpecialType(SpecialType.System_Object)
+                    .WithNullability(typeInfo.Nullability.FlowState);
```

Comment:
hrmm... why is this the right thing to do.  If the typeInfo has no type/convertedType, why are we usin the nullability of it at all.  Is it ever possible to have those both be null, but have 'Nullability' be something reaosnable?

Reply:
You're right, this shouldn't be needed here. The flowstate will end up being Oblivious AFAICT but I haven't looked under the debugger.